### PR TITLE
Preparing 1.1 release

### PR DESCRIPTION
**Pull request by alanz**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,110 @@
...
+- Readme: haskell-ide -> haskell-language-server
+([#1625](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @andys8)
+- Deduplicate main for hie/hie-wrapper
+([#1610](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @Gurkenglas)
+- Azure fix win cabal 8.4.4
+([#1619](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @jneira)
+- Mention the new haskell-ide in the README
+([#1612](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @jneira)
+- Ormolu range format support
+([#1602](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @Av)
```
Comment (fendor) [inline Changelog.md]:
```suggestion
([#1602](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @Avi-D-coder)
```

File: Changelog.md [hunk 2]
```diff
@@ -1,3 +1,110 @@
...
+- Update required stack version
+([#1603](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @wataru86)
+- Remove compiler warnings
+([#1600](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @EncodePanda)
+- Remove hlint.yaml from azure releases and readme
+([#1598](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @jneira)
+- Fix error message parsing to import types
+([#1597](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @fendor)
+- Pass Ormolu cradle flags & default-extensions
+([#1589](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @Av)
```
Comment (fendor) [inline Changelog.md]: same here?

File: Changelog.md [hunk 3]
```diff
@@ -1,3 +1,110 @@
...
+- Update hlint to 2.2.8 and ormolu to 0.0.3
+([#1588](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @alanz)
+- Allow newer optparse-applicative for ormolu
+([#1583](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @bubba)
+- Make cabal-hie-install executable
+([#1581](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @maoe)
+- Ormolu formatter support
+([#1481](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @DavSanchez)
+- GHC 8.8 support
+([#1482](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @Av)
```
Comment (fendor) [inline Changelog.md]: and here?

File: Changelog.md [hunk 4]
```diff
@@ -1,3 +1,110 @@
...
+- cabal now index state 2020-01-31T21:11:24Z
+- GHC 8.8.2 is nightly-2020-01-31
+- GHC 8.6.5 is lts-14.22
+- hlint is 2.2.10
+- brittany is 0.12.1.1
+
+## Changes
+
+- Install script with cabal: check there is one ghc in $PATH
+([#1632](https://github.com/haskell/haskell-ide-engine/pull/1551), by by @jneira)
```
Comment (fendor) [inline Changelog.md]: > by by

a lot of by by

Comment (alanz) [reply Changelog.md]: oops

File: haskell-ide-engine.cabal [hunk 5]
```diff
@@ -1,5 +1,5 @@
 name:                haskell-ide-engine
-version:             1.0.0.0
+version:             1.1
```
Comment (fendor) [inline haskell-ide-engine.cabal]: note sure, shouldnt we still have `1.1.0.0`? doesnt this matter from a versioning scheme?

Comment (alanz) [reply haskell-ide-engine.cabal]: No, it does not matter. It is implicitly 0.0

**Pull request merged**